microapps/FeatureFormsApp/app/src/main/java/com/arcgismaps/toolkit/featureformsapp/screens/map/MapViewModel.kt

```diff
@@ -106,13 +111,25 @@ sealed class UIState {
         val features: Map<String, List<ArcGISFeature>>,
         val featureCount: Int
     ) : UIState()
+
+    /**
+     * Indicates an error state with the given [error].
+     */
+    data class Error(
+        val featureForm: FeatureForm,
+        val title: String,
+        val details: String,
+        val subTitle: String = ""
+    ) : UIState()
 }
 
 /**
  * Class that provides a validation error [error] for the field with name [fieldName]. To fetch
  * the actual message string use [FeatureFormValidationException.getString] in the composition.
```

Comment:
Should this comment say `getMessage`? 

```suggestion
 * the actual message string use [FeatureFormValidationException.getMessage] in the composition.
```